aix/SPECS/3.11.0/wazuh-agent-3.11.0-aix.spec

```diff
@@ -0,0 +1,292 @@
+# Spec file for AIX systems
+Name:        wazuh-agent
+Version:     3.11.0
+Release:     1
+License:     GPL
+URL:         https://www.wazuh.com/
+Vendor:      Wazuh, Inc <info@wazuh.com>
+Packager:    Wazuh, Inc <info@wazuh.com>
+Summary:     The Wazuh agent, used for threat detection, incident response and integrity monitoring.
+
+Group: System Environment/Daemons
+AutoReqProv: no
+Source0: %{name}-%{version}.tar.gz
+Conflicts: ossec-hids ossec-hids-agent wazuh-manager wazuh-local
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+BuildRequires: coreutils automake autoconf libtool
+
+%description
+Wazuh is an open source security monitoring solution for threat detection, integrity monitoring, incident response and compliance.
+
+%prep
+%setup -q
+./gen_ossec.sh init agent %{_localstatedir}/ossec > ossec-init.conf
+cd src && gmake clean && gmake deps RESOURCES_URL=http://packages.wazuh.com/deps/3.9
```

Comment:
This line is downloading the external dependencies from 3.9. This should be 3.11.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/254/commits/bd9469d9104aaced5b40e1a09a527bbe1cefa707